AzureCommunicationCalling.podspec

```diff
@@ -25,40 +25,27 @@
 # --------------------------------------------------------------------------
 
 Pod::Spec.new do |s|
-  s.name = 'AzureSDK'
-  s.version = '1.0.0-beta.2'
-  s.summary = 'Azure SDK for iOS'
-  s.description = <<-DESC
-  Azure SDK for iOS client libraries
-  DESC
+    s.name = 'AzureCommunicationCalling'
+    s.version = '1.0.0-beta.4'
+    s.summary = 'Azure Communication Calling SDK for iOS'
+    s.description = <<-DESC
+    Azure Communication Calling SDK for iOS
+    DESC
+  
+    s.homepage = 'https://github.com/Azure/azure-sdk-for-ios'
+    s.license = { :type => 'Commercial',
+                  :file => 'EULA.txt' }
+    s.authors = { 'Azure SDK Mobile Team' => 'azuresdkmobileteam@microsoft.com' }
+  
+    s.ios.deployment_target = '12.0'
+    s.ios.vendored_frameworks = "AzureCommunicationCalling.framework"
+
+    s.swift_version = '5.0'
+  
+    s.source = { :http => 'https://github.com/Azure/Communication/releases/download/v1.0.0-beta.4/azurecommunicationcalling.framework-1.0.0-beta.4.zip' }
+    s.source_files = 'Headers/*.h'
+    s.public_header_files = "Headers/*.h"
+    s.dependency 'AzureCore', '~> 1.0.0-beta.2'
```

Comment:
I don't believe Calling depends on AzureCore any longer, double check but we should be able to remove this.

Reply:
Dependency needs to be still there for older releases we have done.

Reply:
If the podspec is starting at 1.0.0-beta.4 and 1.0.0-beta.4 doesn't require AzureCore I don't see any reason it couldn't be removed.

Reply:
Yes now i get it, AzureCommunication already specifes deps on AzureCore , so yes we dont directly dont depend on AzureCore